File: public/programs/cpp/p14.cpp
// Arithmetic Progression

#include<iostream.h>
using namespace std;

int main()
{
	int i,n,a,d,last;
	cout<<"ARITHMETIC PROGRESSION....\n";
	cout<<"Enter first term:";
	cin>>a;
	cout<<"\nEnter difference:";
	cin>>d;
	cout<<"\nEnter n:";
	cin>>n;
	for(i=1;i<=n;i++)
	{
		cout<<a<<" ";                      //AP sequence is a,a+d,a+2d.........a+(n-1)d
		a=a+d;
	}
}
	

File: public/programs/java/p14.java
import java.util.*;
class p14
{
	public static void main(String[] args) {
		int a,b,n;
		Scanner scan=new Scanner(System.in);
		System.out.print("Enter n value: ");
		n=scan.nextInt();
		System.out.print("Enter first number: ");
		a=scan.nextInt();
		System.out.print("Enter the difference: ");
		b=scan.nextInt();
		System.out.print("The Arithmetic progression series is ");
		for(int i=0;i<n;i++)
		{
			System.out.print(a+" ");
			a=a+b;
		}
	}
}